Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -2665,6 +2665,8 @@ void FrameSelection::revealSelection(const RevealSelectionOptions& revealSelecti
     if (isNone())
         return;
 
+    m_selectionRevealMode = SelectionRevealMode::DoNotReveal;
```

Comment:
Wouldn't it be more correct to reset after `LocalFrameView::scrollRectToVisible`? There are other early returns before then.

Reply:
Yes, that'll work.